sw/makefile

```diff
@@ -19,38 +21,37 @@ none:
 	@echo "sudo make uninstall - removes pistackmond"
 
 rpi3:
-	make $(EXECS) PLATFORM=RASPBERRY_PI3
+	${MAKE} $(EXECS) PLATFORM=RASPBERRY_PI3
 
 rpi4:
-	make $(EXECS) PLATFORM=RASPBERRY_PI4
+	${MAKE} $(EXECS) PLATFORM=RASPBERRY_PI4
 
 n2:
-	make $(EXECS) PLATFORM=ODROID_N2
+	${MAKE} $(EXECS) PLATFORM=ODROID_N2
 
 c1:
-	make $(EXECS) PLATFORM=ODROID_C1
+	${MAKE} $(EXECS) PLATFORM=ODROID_C1
 
 ${EXECS}: src/pistackmond.cpp
 	${GCC} ${LIBS} ${GCCFLAGS} -D=${PLATFORM} -o ${EXECS} ${SRC} ${LDLIBS}
 
 clean:
-	rm -f ${EXECS}
-
-install: ${EXECS}
-	cp ${EXECS} /usr/local/sbin
-	chown root:root /usr/local/sbin/${EXECS}
-	cp ${SERVICE} ${SERVICEPATH}
-	chown root:root ${SERVICEPATH}/${SERVICE}
-	chmod 644 ${SERVICEPATH}/${SERVICE}
+	rm -f ${EXECS} ${SERVICE}
+
+${SERVICE}: ${SERVICE}.in
+	sed -e "s,PREFIX,${PREFIX}," $< > $@
+
+install: ${EXECS} ${SERVICE}
+	install -p -s ${EXECS} ${PREFIX}/sbin
+	install -p -m 0664  ${SERVICE} ${SERVICEPATH}
 	systemctl daemon-reload
-	systemctl enable pistackmond
-	systemctl start pistackmond
+	-systemctl enable pistackmond
+	-systemctl start pistackmond
```

Comment:
What is the intention of this change? I'm not familiar with this syntax.

Reply:
The minus will handle failure gracefully. If you call install repeatedly without uninstall then the minus helps